cptv-reprocess.py

```diff
@@ -12,21 +12,24 @@ def reprocess(args):
 
     print(f"Querying server {args.server}")
     print(f"Limit is {args.limit}")
-    if len(args.recording_id) == 1:
+    if args.recording_id and len(args.recording_id) == 1:
```

Comment:
if you make the default for `recording_id` an empty list, then this wouldn't be needed everywhere